core/modules/memman/MemMan.h

```diff
@@ -97,7 +97,39 @@ class MemMan {
     static MemMan* create(uint64_t maxBytes, std::string const& dbPath);
 
     //-----------------------------------------------------------------------------
-    //! @brief Lock a set of tables in memory for a particular chunk.
+    //! @brief Lock a set of tables in memory passed to the prepare() method.
+    //!
+    //! The lock() method fails if any REQUIRED table cannot be locked in; as
+    //! prepare() verified such tables could be locked. FLEXIBLE tables that
+    //! had memory reserved are locked but nothing fails if they cannot be.
+    //!
+    //! @param  handle - Handle returned by prepare() given a set of tables.
+    //! @param  strict - When false, if all the required tables are not locked
+    //!                  (i.e. return != 0) the handle is automatically deleted
+    //!                  and any partially locked resources are released.
+    //!                  Otherwise, the handle remains valid and any partially
+    //!                  locked resources remain locked. The handle must be
+    //!                  manually released by calling unlock().
+    //!
```

Comment:
Maybe it's just me, but strict does the opposite of what I expect. Maybe rename it tolerant?